kaolin/graphics/dib_renderer/utils/mesh.py

```diff
@@ -365,51 +454,50 @@ def saveobjscale(meshfile, scale, maxratio, shift=None):
             continue
         else:
             if data[0] == 'v':
-                p = [scale * float(d) for d  in data[1:]]
+                p = [scale * float(d) for d in data[1:]]
                 meshfp2.write('v %f %f %f\n' % (p[0], p[1], p[2]))
             else:
                 meshfp2.write(line)
                 continue
-                
+
     meshfp.close()
     meshfp2.close()
-    
+
     return
 
 
-################################################################3    
+# 3
```

Comment:
What is this commented out 3 ?

Reply:
My guess was that the original code had 3 as a typo ("#" is shift+3), and the auto formatter removed all extra "#"s